lib/shroom/pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+
+[tool]
+
+[tool.black]
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+)/
+'''
+include = '\.pyi?$'
+line-length = 100
+target-version = ['py36', 'py37', 'py38', 'py39']
+
+[tool.coverage.report]
+exclude_lines = [
+  "if __name__ == .__main__.:",
+  "pass",
+  "pragma: no cover",
+  "raise NotImplementedError",
+  "raise TypeError",
+  "raise ValueError"
+]
+
+[tool.isort]
+known_local_folder = [
+  'style-transfer',
+  'tests'
+]
+line_length = 100
+profile = 'black'
+
+[tool.poetry]
+authors = ["Arkhn's AI Team <ai@arkhn.com>"]
+description = "a package to experiement style transfer using RLF, LLM, ReST ?"
+license = "Apache-2.0"
+name = "style-transfer"
+readme = "README.md"
+repository = "https://github.com/arkhn/open-nlp"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+datasets = "^2.16.1"
+hydra-colorlog = "^1.2.0"
+hydra-core = "^1.3.2"
+hydra-joblib-launcher = "^1.2.0"
+langchain = "^0.3.0"
+langchain-community = "^0.3.0"
+langchain-core = "^0.3.0"
+langchain-groq = "^0.2.0"
+pandas = "^2.2.0"
+python = "~3.11"
+rich = "^13.5.2"
+tqdm = "^4.66.1"
+wandb = "^0.16.3"
```

Comment:
the dependencies should be cleared just leave python for now

Reply:
thanks for pointing out